universidade-do-porto-faculdade-de-engenharia-chicago-pt.csl

```diff
@@ -0,0 +1,731 @@
+        </if>
+        <else-if type="article-journal bill book chapter legal_case legislation motion_picture paper-conference" match="none">
+          <text macro="archive"/>
+        </else-if>
+      </choose>
+      <choose>
+        <if type="webpage post-weblog" match="any">
+          <date form="text" variable="issued"/>
+        </if>
+      </choose>
+      <choose>
+        <if match="none" type="song article-journal article article-magazine">
+          <group delimiter=" ">
+            <text term="accessed" text-case="capitalize-first"/>
+            <date form="text" variable="accessed" prefix=""/>
+          </group>
+        </if>
+      </choose>
+      <choose>
+        <if type="legal_case" match="none">
+          <choose>
+            <if variable="DOI">
+              <text variable="DOI" prefix="https://doi.org/"/>
+            </if>
+            <else>
+              <text variable="URL"/>
+            </else>
+          </choose>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if variable="title" match="none">
+        <choose>
+          <if type="personal_communication" match="none">
+            <text variable="genre" text-case="capitalize-first"/>
+          </if>
+        </choose>
+      </if>
+      <else-if type="bill book graphic legislation motion_picture song" match="any">
+        <text variable="title" text-case="title" font-style="italic"/>
+        <group prefix=" (" suffix=")" delimiter=" ">
+          <text term="version"/>
+          <text variable="version"/>
+        </group>
+      </else-if>
+      <else-if variable="reviewed-author">
+        <choose>
+          <if variable="reviewed-title">
+            <group delimiter=". ">
+              <text variable="title" text-case="title" quotes="true"/>
+              <group delimiter=", ">
+                <text variable="reviewed-title" text-case="title" font-style="italic" prefix="Review of "/>
+                <names variable="reviewed-author">
+                  <label form="verb-short" text-case="lowercase" suffix=" "/>
+                  <name and="text" delimiter=", "/>
+                </names>
+              </group>
+            </group>
+          </if>
+          <else>
+            <group delimiter=", ">
+              <text variable="title" text-case="title" font-style="italic" prefix="Review of "/>
+              <names variable="reviewed-author">
+                <label form="verb-short" text-case="lowercase" suffix=" "/>
+                <name and="text" delimiter=", "/>
+              </names>
+            </group>
+          </else>
+        </choose>
+      </else-if>
+      <else-if type="legal_case interview patent webpage" match="any">
+        <text variable="title" quotes="false" prefix="&quot;" suffix="&quot;"/>
+      </else-if>
+      <else>
+        <text variable="title" text-case="title" quotes="false" prefix="&quot;" suffix="&quot;"/>
```

Comment:
The hard coded quotation marks aren't good. This should be handled using terms and quotes="true". -- also, do you actually want straight quotes? I'd expect typographic ones to be preferable

Reply:
You can localize the quotes in the term section and should do that (check the locale files on GitHub to see how). It behaves better than hardcoded quotes.

For the months, if this works in the preview here, that would be a Mendeley bug and you would need to ask them

Reply:
Hi @adam3smith 
The suggestion that you made about > localize the quotes in the term section, works like a charm 👍 
In my locale readings in GitHub I have found some instructions about locale months and I have inserted it in the file. This way is better than the default translation because in Portugal we had an orthographic change, and since them our months are in lowercase.
Thanks for the help ;)

Reply:
If the lowercasing of months is generally the case now for pt-PT, we'd welcome a pull request on the PR.

Reply:
Hi @adam3smith, Yes the months are now (for some years) in lowercase in Portuguese - Portugal. And its in lowercase as well in Portuguese - Brasilian.
Please **tell me where is the file** to make a pull request and I'll be happy to help improve the portuguese translation. And all the improvements I will suggest, they will be suitable for all portuguese language countries.